File: day 48/homework/code/codewars8.py

"""https://www.codewars.com/kata/56606694ec01347ce800001b/train/python"""

# Is this a triangle?

def is_triangle(a, b, c):
    if a > 0 and b>0 and c >0 and b+c > a and a+c > b and a+b > c:
        return True
    return False

File: day 080/homework/code/codewars8.js

// https://www.codewars.com/kata/520b9d2ad5c005041100000f/train/javascript

// Simple Pig Latin

function pigIt(str) {
  return str
    .split(' ')
    .map(word => {
      if (/^[a-zA-Z]+$/.test(word)) {
        return word.slice(1) + word[0] + 'ay';
      }
      return word; // punctuation
    })
    .join(' ');
}
